ubuntudesign/www.ubuntu.com
# update marketo country and state field and js

## Done

* rewrote marketo's js that shows/hides the state/province fields based on what you select in the country select dropdown
* added a shared _state.html file as not all forms require state
* removed the old js from the pages
* added country to the /download/cloud ebook form per Sales (Shelley) request as phone numbers often don't have a country code

## QA

1. check that /static/js/forms.js looks ok
2. check that these forms all work as expected
- [x] check /about/contact-us/form
- [x] check /cloud/contact-us
- [x] check /cloud/openstack/managed-cloud/contact-us
- [x] check /cloud/training/onsite-contact-us
- [x] check /cloud/training/server-admin-onsite-contact-us
- [x] check /desktop/tenders-contact-us
- [x] check /download/cloud
- [x] check /internet-of-things/contact-us
- [x] check /internet-of-things/digital-signage/contact-us
- [x] check /management/contact-us
- [x] check /server/contact-us
- [x] check /tablet/contact-us

## Issue / Card

Partly fixes #851
